corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -316,6 +316,9 @@ <h3 data-bind="text: $root.activeCaseType()" style="display: inline-block;"></h3
               </div>
             {% endif %}
             {% if not request.is_view_only %}
+              <div class="row-item-small" data-bind="hidden: containsCaseData() || isGeoCaseProp()">
```

Comment:
@zandre-eng 
Why shouldn't we be able to delete geo case properties?

Reply:
This is just for the active geo case property that is being used by the domain. A user is free to delete other geo case properties that aren't linked to any case data. This is just a safety precaution so that the user does not accidently delete the property which is being used by the geospatial feature. If they would like to delete it, then the user will need to first change the domain's active geo case property.